code/modules/jobs/job_types/_job.dm

```diff
@@ -346,6 +354,10 @@
 	types += duffelbag
 	return types
 
+// An overridable getter for more dynamic goodies.
+/datum/job/proc/get_mail_goodies(mob/recipient)
+	return mail_goodies
```

Comment:
What is the point of this proc? Why does it need an arg anyway? You don't need getters here.